lib/autokey/iomediator/__init__.py

```diff
@@ -1,23 +0,0 @@
-# Copyright (C) 2018 Thomas Hess
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-"""
-IoMediator public interface
-"""
-
-from ._windowgrabber import WindowGrabber
```

Comment:
Writing this file at all was a mistake. These imports caused even more tight coupling and circular imports. Now, the code just imports directly from the submodules.